src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
-// write your code here
+const columns = document.querySelectorAll('tr');
+
+columns.forEach((el) => {
+  const posColumn = el.cells[1];
+  const posColumnClone = posColumn.cloneNode(true);
+  const lastColumn = el.cells[el.cells.length - 1];
+
+  el.insertBefore(posColumnClone, lastColumn);
```

Comment:
The insertBefore method is used incorrectly. The second parameter should be a reference node, which is the node before which the new node will be inserted. In this case, it should be the node after 'lastColumn', which is 'null' since 'lastColumn' is the last element.